code/__linters.dm

```diff
@@ -10,3 +10,25 @@ Nothing should ever be included before this file.
 #ifdef OPENDREAM
 #include "___linters\odlint.dm"
 #endif
+
+
+//Poor man's code to try to catch people that are running the codebase without compiling tgui, aka in an unsupported way
+//Basically, this tells you to use a supported compilation as defined in the repo's README
+#if DM_VERSION < 515
+/world/New()
+	if(!fexists("./tgui/public/tgui.bundle.js"))
+		world.log << "RTFM Error ID: 10-T"
```

Comment:
hilarious
how about instead we write an actually informative error message though

Reply:
The message is informative: Read The Fucking Manual, and it also works perfectly for what I wish it to do: Tell us that someone didn't read the instructions, because while this is a critical error, others will soon follow (eg: configuration missing)